<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="Philomina" failures="1" tests="3" name="Entrata Test" time="87.552" errors="0" timestamp="2025-07-06T21:30:29 IST">
  <testcase classname="Entrata.testcases.FirstcaseTest" name="verifyEntrataHomepage" time="4.907">
    <failure type="java.lang.AssertionError" message="expected [https://www.entrata.com/press/blackstone] but found [https://www.entrata.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://www.entrata.com/press/blackstone] but found [https://www.entrata.com/]
at pages.firstcasepage.companyNews(firstcasepage.java:72)
at Entrata.testcases.FirstcaseTest.verifyEntrataHomepage(FirstcaseTest.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyEntrataHomepage -->
  <testcase classname="Entrata.testcases.SecondcaseTest" name="verifyResourcespage" time="48.791"/>
  <testcase classname="Entrata.testcases.ThirdcaseTest" name="verifyDemoform" time="4.462"/>
  <testcase classname="Base.EntrataBase" name="@AfterTest teardown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase classname="Base.EntrataBase" name="@AfterTest teardown" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest teardown -->
  <testcase classname="Base.EntrataBase" name="@BeforeTest launchDriver" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest launchDriver -->
  <testcase classname="Base.EntrataBase" name="@BeforeTest launchDriver" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest launchDriver -->
</testsuite> <!-- Entrata Test -->
